test/run.sh

```diff
@@ -59,6 +60,22 @@ expect "sftp>"
 send "put ${upload_id}.tgz ci/travis/\r"
 expect "sftp>"
 send "bye\r"
+EOF
+    cd -
+  fi
+# upload github actions log if test failed
+  if [ -n "${BUILD_BUILDID:-}" ] && [ -n "${LOGBAK_SERVER:-}" ]; then
+    upload_id="github-actions-${test_id}-${BUILD_BUILDID:-0}-${ImageOS:-unknown}"
+    cd "${test_tmp_dir}"/..
+    tar -czf "${upload_id}.tgz" "${test_id}"
+    expect <<EOF
+spawn sftp -o "StrictHostKeyChecking=no" "${LOGBAK_USER}@${LOGBAK_SERVER}"
+expect "assword:"
```

Comment:
```suggestion
expect "password:"
```

Reply:
`expect "assword:"` is better.

Some implementations are `Password:` and others are `password:`.

Reply:
Got a tip